<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.yg.laboratory.dao.UserDaoTest" tests="4" skipped="0" failures="0" errors="0" timestamp="2024-07-01T06:44:47" hostname="bag-yeong-gyuui-MacBookPro.local" time="0.533">
  <properties/>
  <testcase name="getUserFailure()" classname="com.yg.laboratory.dao.UserDaoTest" time="0.49"/>
  <testcase name="andAndGet()" classname="com.yg.laboratory.dao.UserDaoTest" time="0.017"/>
  <testcase name="count()" classname="com.yg.laboratory.dao.UserDaoTest" time="0.018"/>
  <testcase name="userDao_test()" classname="com.yg.laboratory.dao.UserDaoTest" time="0.005"/>
  <system-out><![CDATA[데이터 베이스 연결중1
테스트중!
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
